ClipboardAPI/tag-security-privacy-clipboard-svg.md

```diff
@@ -0,0 +1,97 @@
+### Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature exposes `image/svg+xml` MIME type to the clipboard using the async clipboard API to support copy-paste of SVG images. All the restrictions related to secure context, permission etc apply for reading this format. The format is parsed using the strict HTML processing that strip out security sensitive tags.
```

Comment:
```suggestion
This feature exposes the `image/svg+xml` MIME type to the clipboard using the async clipboard API to support copy-paste of SVG images. All the restrictions related to secure context, permission etc apply for reading this format. The format is parsed using strict HTML processing that strips out security-sensitive tags.
```

Grammar nits.

Reply:
Also -- is there anything more specific that can be said about the sanitization? From [comments](https://docs.google.com/document/d/1jq8QSCQRdNy99rnPusmW8is62c22PVuq-Sk-tMT2tRk/edit?disco=AAAAGzUW4fQ) in the design doc, it sounds like this will use the same sanitization as HTML put on the clipboard, but that there may also be some SVG-specific additions needed. Do you know what we plan to do there?

Reply:
Yeah, I also noticed that we're using the strict HTML markup processing for SVG image payload to strip out `<script>` tags and inline styles into the markup which is similar to how we process HTML content. I think it needs more discussion and I agree with that comment that this probably needs an [XML parser to parse the SVG document](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser/parseFromString) and remove security-sensitive tags using a different processing. Inlining styles and removing the `<style>` elements can also lead to loss of formatting information which would be detrimental to SVG images.

Reply:
@snianu and I spoke separately -- we're going to leave this unmerged while more details of how sanitization will be handled are worked out.